### ISSUE TITLE
Does the pretrained Relaxer expect primitive or conventional unit cells for the structures?

### ISSUE THREAD

**Issue by sgbaird** [materialsvirtuallab/m3gnet#10]
I'm planning to use the pretrained model per the usage instructions, and I'm wondering if I should make sure to convert to primitive or conventional unit cells before passing to the relaxer (https://github.com/sparks-baird/xtal2png/blob/7386edd9eb0748d1f31bf6e160b18f5dc2b05107/src/xtal2png/core.py#L862-L869)

As a follow-up to my poorly phrased question in #9

**Comment by shyuep** [comment]
The model was trained on MP relaxations, which are mostly the primitive cells. However, like all IAPs, what matters is whether it captures the relationship between energies/forces and the atomic environments. I wouldn't think it matters whether you use primitive or conventional cells. For the benchmarks I showed, I used the conventional cubic cells because that is the easiest for most people to interpret. It appears just as accurate even though the original model was trained on primitive cells (mostly).
